### ISSUE TITLE
Add the subcommand "farg test" 

### ISSUE THREAD

**Issue by amahabal** [amahabal/PySeqsee#31]
Currently, I use nosetests, and do the following:

nosetests --pdb tests/ tests/seqsee

It would be nice to have "farg test", and best if it picked up files in the current directory instead of installed files.

**Comment by superMDguy** [comment]
I'll work on that.

Also, I was thinking it'd be nice to have commands that can be executed in the project directory, so typing out the name of the project isn't necessary.  I'll try to get that working as well.

**Comment by amahabal** [comment]
Awesome, I am glad that you are here 👍

One aspect that bothered me with the tests was that it was picking up, I
think, the globally installed farg modules rather than those in the current
directory. This new command would help avoid that error.

While you are at it, could you also move tests from tests/seqsee to
farg/apps/seqsee/tests?

On Sat, Jan 14, 2017, 6:17 AM Matthew D. <notifications@github.com> wrote:

I'll work on that.

Also, I was thinking it'd be nice to have commands that can be executed in
the project directory, so typing out the name of the project isn't
necessary. I'll try to get that working as well.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/amahabal/PySeqsee/issues/31#issuecomment-272626877>,
or mute
the thread

**Comment by superMDguy** [comment]
I moved around the

**Comment by superMDguy** [comment]
I moved around the test folders a little bit (See commit 0bdcb53), and I was just about to add the `farg test` command.  I just wanted to make sure I got it right.  Did you want the command to execute tests in the project in the current directory (for example running `nosetests --pdb ./tests`), or did you want it running the tests in a specified project (like `farg test seqsee`)?  Or, I could make both work.
